src/main/java/com/maxmind/minfraud/request/PaymentProcessorsList.java

```diff
@@ -0,0 +1,168 @@
+package com.maxmind.minfraud.request;
+
+/**
+ * Enumeration of payment processors
+ */
+public enum PaymentProcessorsList {
```

Comment:
This is a breaking change. We do not want to move this.

Maybe we could disable the missing doclint in the Maven plugin and use checkstyle to enforce the inclusion of javadocs. I think with that, we could disable the warning on enums only.